### PR TITLE
let's you replace your arm with an energy chainsaw without it being shit

### PR DESCRIPTION
**Pull request by xTrainx**
[Changelogs]: # 
:cl: xTrainx
add: Added mounted energy chainsaw
tweak: putting a chainsaw in your arm now distinguish betwen energy chainsaw and normie chainsaw
/:cl:

[why]: # fixes issue #4510

### CODE REVIEW

File: hippiestation/code/game/objects/items/weaponry.dm [hunk 1]
```diff
@@ -46,4 +46,22 @@
 			H.set_heartattack(TRUE)
 			if(H.stat == CONSCIOUS)
 				H.visible_message("<span class='userdanger'>[H] clutches at [H.p_their()] chest as if [H.p_their()] heart stopped!</span>")
-			
+		
+/obj/item/mounted_energy_chainsaw
```
Comment (ghost) [inline hippiestation/code/game/objects/items/weaponry.dm:50]: Code doesn't work, not properly indented.

File: code/modules/surgery/prosthetic_replacement.dm [hunk 2]
```diff
@@ -78,11 +78,15 @@
 		user.visible_message("[user] finishes attaching [tool]!", "<span class='notice'>You attach [tool].</span>")
 		qdel(tool)
 		if(istype(tool, /obj/item/twohanded/required/chainsaw))
-			var/obj/item/mounted_chainsaw/new_arm = new(target)
-			target_zone == "r_arm" ? target.put_in_r_hand(new_arm) : target.put_in_l_hand(new_arm)
-			return 1
+			if(istype(tool, /obj/item/twohanded/required/chainsaw/energy))
```
Comment (ghost) [inline code/modules/surgery/prosthetic_replacement.dm]: You are adding new code to /tg/ code file
Either mirror and update the proc in the hippiestation subfolder or add a comment saying the start and end of hippiecode if that is not feasible.

**Review by ghost — CHANGES_REQUESTED** [review]
Indent everything under line 50 in weaponry.dm

File: hippiestation/code/game/objects/items/weaponry.dm (hunk 1 above)
Comment (xTrainx) [reply hippiestation/code/game/objects/items/weaponry.dm:50]: Sorry, i have autism

**Comment by xTrainx** [comment]
and it only took me 6 commits

**Comment by 2Dollarbill** [comment]
Why not, I think it'd be fine, more toys for the boys. +1

**Comment by GuyonBroadway** [comment]
reminder that you need two hands just to hold a chainsaw so without a willing helper this is impossible to do yourself... and that's fine IMO.

File: code/modules/surgery/prosthetic_replacement.dm [hunk 3]
```diff
@@ -78,11 +78,15 @@
 		user.visible_message("[user] finishes attaching [tool]!", "<span class='notice'>You attach [tool].</span>")
 		qdel(tool)
 		if(istype(tool, /obj/item/twohanded/required/chainsaw))
-			var/obj/item/mounted_chainsaw/new_arm = new(target)
```
Comment (Carbonhell) [inline code/modules/surgery/prosthetic_replacement.dm:81]: Why not just change this from new(src) to new tool(src)?

File: hippiestation/code/game/objects/items/weaponry.dm (hunk 1 above)
Comment (Carbonhell) [reply hippiestation/code/game/objects/items/weaponry.dm:50]: Maybe make it a child of the normal mounted chainsaw?

**Review by Carbonhell — CHANGES_REQUESTED** [review]

**Comment by Carbonhell** [comment]
This isnt really a feature but a bugfix. Its passable since the system beneath it is shitty. A rework in future could be wise. Merging

**Review by Carbonhell — APPROVED** [review]

**Pull request merged**